kolena/workflow/_validators.py

```diff
@@ -52,6 +52,8 @@ def validate_data_object_type(
     data_object_type: Type[DataObject],
     supported_field_types: Optional[List[Type]] = None,
     supported_list_types: Optional[List[Type]] = None,
+    supported_dict_key_types: Optional[List[Type]] = [],
+    supported_dict_value_types: Optional[List[Type]] = [],
```

Comment:
Is there a reason why this is `[]` while the others are `None`?

Reply:
In general, default mutable objects should be avoided in python: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Yes I miss that one. Updated.